### PR TITLE
Fix unsignedness issue

### PR DESCRIPTION
**Pull request by AtariDreams**
H.

### CODE REVIEW

File: notifier/rss.c [hunk 1]
```diff
@@ -385,8 +385,8 @@ main(int  argc,				/* I - Number of command-line arguments */
 
   if (http)
   {
-    unlink(filename);
     httpClose(http);
+    unlink(filename);
```
Comment (michaelrsweet) [inline notifier/rss.c:389]: I don't see the point of this change.

**Review by michaelrsweet — CHANGES_REQUESTED** [review]
Aside from the rss.c change, this looks good.

File: notifier/rss.c (hunk 1 above)
Comment (AtariDreams) [reply notifier/rss.c:389]: Consistency with the other block of code that does this.

**Comment by AtariDreams** [comment]
@michaelrsweet I should have sent this privately instead of making it public since this is a security issue that goes back years. Depending on its severity we may have to ship this patch as soon as possible.

**Comment by AtariDreams** [comment]
I sent Apple a security report just in case.

**Pull request closed without merging**